.env.development

```diff
@@ -1,6 +1,7 @@
 ADMIN_HOST='http://localhost:3000'
 API_SERVICE_BACKEND_URL_OPTIONS={"uk":"http://localhost:3000","xi":"http://localhost:3000" }
 AUTHENTICATE_WITH_SSO=false
+BASIC_PASSWORD=dev123
```

Comment:
Don't we need to add this to the aws secret manager?

Reply:
This is just for local development and yeah I've added a different one for staging and development aws accounts